journal/original/29-chapter-28.adoc

```diff
@@ -1,7 +1,8 @@
-== Section XXVIII.
+== Chapter XXVIII
 
-In several meetings, both National and Provincial, he loas earnest to have marriage,
-which is God`'s ordinance, both begun and accomplished in God`''`s counsel.
+[.chapter-synopsis]
+* In several meetings, both National and Provincial, he was earnest to have marriage,
+* which is God`'s ordinance, both begun and accomplished in God`'s counsel
```

Comment:
Oops, it looks like this is one of those times where the linter added an asterisk before a newline that shouldn't have been there. I've made a note to look into that behavior and change it possibly.  Can you quick fix this and make it match the original edition? It should all be one line.